pyfar/plot/_line.py

```diff
@@ -462,15 +462,22 @@ def _deal_time_units(unit='s'):
 def _spectrogram(signal, dB=True, log_prefix=20, log_reference=1,
                  yscale='linear', unit=None,
                  window='hann', window_length=1024, window_overlap_fct=0.5,
-                 cmap=mpl.cm.get_cmap(name='magma'), ax=None):
+                 cmap=mpl.cm.get_cmap(name='magma'), colorbar=True, ax=None):
     """Plot the magnitude spectrum versus time.
 
     See pyfar.line.spectogram for more information.
+
+    Note: this function always returns only the axis of the actual plot and
+    never an array of axes containing also the axis of the colorbar. This makes
+    handling interactions easier. The axis of the colorbar is added in
+    pyfar.line.spectrogram.
```

Comment:
This is a little confusing as ax[0], qm, cb are returned. Could you reformulate it to explain what these 3 objects are? :)

Reply:
It is documented in the public function - can you check if you find the information there sufficient?

Reply:
```suggestion
    Note: this function always returns only the axis of the actual plot together with the quadmesh and colorbar, and
    never an array of axes containing also the axis of the colorbar. This makes
    handling interactions easier. The axis of the colorbar is added in
    pyfar.line.spectrogram.
```
It think, then it's clear :)

Reply:
ok added this :)